src/fragments/lib/auth/native_common/getting_started/common.mdx

```diff
@@ -39,6 +46,8 @@ Enter the following when prompted:
     `No, I am done.`
 ```
 
+> If you have previously enabled an Amplify category that uses Auth behind the scenes (e.g. API category), run the `amplify update auth` command to edit your configuration. 
```

Comment:
Ditto could consider to use the `Callout` block.

Reply:
Valid point.  I've updated all references to this to be within a `<Callout>` now.  Let me know if this looks better. 

CC: @abdallahshaban557

Reply:
Reverting these back after reviewing with PM.